airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "orchestrator"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
❗ Not really required, but good hygiene. You should run `poetry update metadata_service` from the orchestrator package to update the lock file.